src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import ViewMenu from "./components/ViewMenu";
+
+const App = () => {
+  return (
+    <>
+      <h1>Turtle - Food fast</h1>
+      <ViewMenu />
```

Comment:
Why name it `ViewMenu` and not just `Menu`?

Reply:
ok, we can live with that!